theano/gof/type.py

```diff
@@ -603,26 +633,32 @@ class CDataType(Type):
         The type of the pointer (complete with the `*`).
 
     freefunc
-        A function to call to free the pointer. This function must have a `void`
-        return and take a single pointer argument.
+        A function to call to free the pointer. This function must
+        have a `void` return and take a single pointer argument.
 
     """
 
-    import ctypes
-    if PY3:
-        _cdata_type = ctypes.py_object.from_address(
-            ctypes.addressof(ctypes.pythonapi.PyCapsule_Type)).value
```

Comment:
As we are droping python 2.6, I think we can clean up the code to replace PyCObject with PyCapsule usage everywhere. Can you confirm this? I make a note about this in the issue about dropping python 2.6